### DISCUSSION TITLE
see the image , how to change the tab color and border color ? how to hide "Zellij(wide-bushes)"

### DISCUSSION THREAD

**Discussion by wacdev** [zellij-org/zellij#2303] [Q&A]
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/127726508/226103404-f01d398e-b668-47d0-bfb0-5f5b725c9bfe.png">

I want config this like my tmux as below
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/127726508/226103556-6d505bd8-c7fb-4b82-80b1-08d0a0ebf420.png">

**Comment by mark2185** [comment]
The things that [the docs](https://zellij.dev/documentation/themes.html) suggest didn't work?

**Reply by wacdev** [reply]
How to configure these is not stated in the themes document? I only see how to set presets for each color
